projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
WTF?! The old PR was open for some days and anybody could have said that the copyright is not okay. After the PR was closed the copyright thing was faulted and I did what was proposed.